diagnostic.md

```diff
@@ -5,74 +5,83 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+The purpose of a back end is to connect clients to servers and store data.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+The model is used by the controller to fetch data.
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+The controller, which contains the business logic communicates with the model.
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+We don't use views because we have single page applications with only one piece of markup.
```

Comment:
We don't use views as a part of our back-end because we can handle managing view states on the front end.